server/models.py

```diff
@@ -2,23 +2,63 @@
 from flask_sqlalchemy import SQLAlchemy
 from flask_marshmallow import Marshmallow
 from marshmallow import Schema, fields, pre_load, validate
-
+from typing import List
 
 # Order matters: Initialize SQLAlchemy before Marshmallow
 db = SQLAlchemy()
 ma = Marshmallow()
 
+association_table = db.Table('association_table',
```

Comment:
you want this table to have a name like "user_skills"

Reply:
👍